lib/cl/externals/software/usocket-0.7.0.1/vendor/OpenTransportUDP.lisp

```diff
@@ -0,0 +1,146 @@
+;;;-*-Mode: LISP; Package: CCL -*-
```

Comment:
As with the other libraries, this should not be checked in.  I have skipped over all of the externals/ due to this.